Source Code/Front-End/UI (WPF)/Smart Pacifier - Tool/Smart Pacifier - Tool/Tabs/SettingsTab/SettingsView.xaml

```diff
@@ -1,12 +1,91 @@
-﻿<UserControl x:Class="Smart_Pacifier___Tool.Tabs.SettingsTab.SettingsView"
-             xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
-             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
-	<Grid>
-		<!-- To be removed-->
-		<TextBlock Text="Welcome to the Settings View"
-                   HorizontalAlignment="Center"
-                   VerticalAlignment="Center"
-                   FontSize="24"
-                   FontWeight="Bold"/>
-	</Grid>
-</UserControl>
\ No newline at end of file
+﻿<UserControl x:Class="Smart_Pacifier___Tool.Tabs.SettingsTab.SettingsView"
+             xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:av="http://schemas.microsoft.com/expression/blend/2008"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+             mc:Ignorable="av" Width="538" Height="507">
+
+    <UserControl.Resources>
+        <!-- Modern Button Style with Rounded Corners, Hover Effect, and Shadows -->
+        <Style x:Key="ModernButtonStyle" TargetType="Button">
+            <Setter Property="Background" Value="#0000" />
+            <Setter Property="Foreground" Value="White" />
+            <Setter Property="Padding" Value="15,7" />
+            <Setter Property="FontWeight" Value="Bold"/>
+            <Setter Property="BorderBrush" Value="#FFFF" />
+            <Setter Property="BorderThickness" Value="1" />
+            <Setter Property="FontSize" Value="14"/>
+            <Setter Property="Height" Value="60"/>
+            <Setter Property="Template">

```

Comment:
Can you define this style globally so that we can reuse it, and if we need to change the style, we'll only have to do it in one place?

Reply:
Thats a good idea, will do it in the morning. Thanks :)